### ISSUE TITLE
For "dumbphone" users, bridge SMS interface to incident chat room

### ISSUE THREAD

**Issue by fabacab** [betterangels/buoy#14]
For an alerter: an ability to txt `HELP` or some other similar command to an SMS-capable phone number that Buoy can receive.

For a responder: the ability to respond `RESPOND` or some other similar command to an SMS-capable phone number that Buoy can receive.

When a responder responds via SMS, forward messages from the chat room to the responder's SMS number. Forward any further messages sent from the responder's dumbphone via SMS into the chat room for the other responders to read. The special command `ALERT STOP` or something can leave the chatroom.

Basically, a bridge between the smartphone-capable interface and an SMS-only interface should eventually be implemented to support anyone who does not have access to a smartphone.

**Comment by fabacab** [comment]
The default txt notification scheme currently uses telco provider's email-to-SMS gateways. Can we reverse the process and have administrators create an email address for the Buoy app itself (like `buoy@example.com`) to which registered users can send an SMS. We then have Buoy automatically monitor (poll?) that email address for commands and executes in small batches? (Can use [the `crontab-manager.php` code already written](https://github.com/meitar/better-angels/blob/ae51f59ad36a234aa7529c88c5e5f5834f476a81/includes/crontab-manager.php) for the scheduled alerts for this purpose.)

**Comment by fabacab** [comment]
This feature idea is a good candidate for extraction into its own plugin. Rather than write this functionality specifically for Buoy, we should write it as its own plugin that simply makes the WordPress (admin) API available via txt bridge and then we should hook our plugin's functionality into that other plugin where needed.
